src/MICore/Debugger.cs

```diff
@@ -978,6 +996,28 @@ public void OnDebuggerProcessExit(/*OPTIONAL*/ string exitCode)
             }
         }
 
+        string GdbVersionFromLog()
+        {
+            foreach (string line in _initializationLog)
+            {
+                // Second set of parenthesis looks for a Cygwin-specific version number
+                // Cygwin example: GNU gdb (GDB) (Cygwin 7.11.1-2) 7.11.1
+                // MinGW example:  GNU gdb (GDB) 8.0.1
+                Match match = Regex.Match(line, "GNU gdb \\(GDB\\) (?:\\(Cygwin (\\d+[\\d\\.-]*)\\) )?(\\d+[\\d\\.-]*)");
+                if (match.Success)
+                {
+                    return match.Groups[1].Success ? match.Groups[1].Value : match.Groups[2].Value;
+                }
+            }
+
+            return null;
+        }
+
+        bool IsUnsupportedGdbVersion(string version)
```

Comment:
can we change this to be IsUnsupportedWindowsGdbVersion or add a comment that this is only the unsupported versions running on a Windows subsystem?